apps/credential-service/lib/server.ts

```diff
@@ -35,7 +35,7 @@ assert(process.env.GITHUB_APP_ID, 'GITHUB_APP_ID is not defined in the environme
 const GITHUB_APP_ID = process.env.GITHUB_APP_ID;
 /** The PEM key of the Github app used to generating tokens. */
 assert(process.env.GITHUB_APP_PEM, 'GITHUB_APP_PEM is not defined in the environment');
-const GITHUB_APP_PEM = process.env.GITHUB_APP_PEM;
+const GITHUB_APP_PEM = Buffer.from(process.env.GITHUB_APP_PEM, 'base64').toString('utf-8');
```

Comment:
Any reason to encode the secret as base64 in the environment? Please clarify in code/or here, and also rename the environment variable to e.g. `GITHUB_APP_PRIVATE_KEY_BASE64`

Reply:
Done